app/test_main.py

```diff
@@ -1,3 +1,70 @@
 from app.main import get_human_age
+import pytest
+from typing import Any
 
-# write your code here
+
+class TestValues:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected_value",
+        [
+            pytest.param(
+                15,
+                15,
+                [1, 1],
+                id="Correct values for 15, 15"),
+            pytest.param(
+                24,
+                24,
+                [2, 2],
+                id="Correct values for, 24, 24"),
+            pytest.param(
+                28,
+                29,
+                [3, 3],
```

Comment:
The expected output `[3, 3]` for inputs `28` and `29` seems incorrect based on the conversion rules provided in the task description. According to the rules, `28` cat years should convert to `3` human years, but `29` dog years should convert to `2` human years. Please verify and correct the expected output.